pkg/manifests/types.go

```diff
@@ -376,6 +379,18 @@ type NodeExporterCollectorTcpStatConfig struct {
 	Enabled bool `json:"enabled,omitempty"`
 }
 
+// The `NodeExporterCollectorSysctlConfig` resource works as an on/off switch for
+// the `sysctl` collector of the `node-exporter` agent.
+// By default, the `sysctl` collector is disabled.
```

Comment:
There should be a clear message that if misused, it can screw up Prometheus. For instance if you decide to expose `kernel.random.uuid` via `--collector.sysctl.include-info`, it will generate new series at every scrape.